docs/source/installation.md

```diff
@@ -73,21 +73,46 @@ It should download a pretrained model then print something like
 
 ## Installing from source
 
-To install from source, clone the repository and install with the following commands:
+Note that the following will will install not the latest released version, but the bleeding edge `master` version, which may be not-quite ready for normal usage, but you may want to use it in case a bug has been fixed since the last official release and a new release hasn't yet been rolled out.
+
+```
+pip install git+https://github.com/huggingface/transformers
+```
+
+Again, you can run
+
+```bash
+python -c "from transformers import pipeline; print(pipeline('sentiment-analysis')('I hate you'))"
+```
+
+to check 🤗 Transformers is properly installed.
+
+## Editable install
+
+If you are an advanced user who wants to constantly use the bleeding edge `master` version of the source code, the easiest way to do it is via an editable install. This is done by cloning the repository and installing with the following commands:
 
 ``` bash
 git clone https://github.com/huggingface/transformers.git
 cd transformers
 pip install -e .
 ```
 
-Again, you can run 
+This command performs a magical link between the folder you cloned the repository to and your python library paths, so it'll look inside this folder in addition to the normal library-wide paths. So if normally your python packages get installed into:
```

Comment:
```suggestion
This command performs a magical link between the folder you cloned the repository to and your python library paths, and it'll look inside this folder in addition to the normal library-wide paths. So if normally your python packages get installed into:
```
Nit- avoid the repetition of so.

Reply:
I so totally agree ;) 

This is a good reminder - thank you!